bin/migrate-org-to-scheming.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+"""
+Generates a JSON lines file which can be used to migrate existing
+organizations to use schema defined in organization.yaml
+
+Usage:
+migrate-org-to-scheming.py
+    -s/--server http://registry.open.canada.ca
+    -o/--output outfile.jsonl
+"""
+
+import sys
+import json
+import codecs
+import getopt
+from ckanapi import RemoteCKAN
+
+
+def main(argv):
+    try:
+        server = ''
+        outfile = ''
+        opts, args = getopt.getopt(argv, "s:o:", ["server=", "output="])
+        for opt, arg in opts:
+            if opt in ['-s', '--server']:
+                server = arg
+            if opt in ['-o', '--output']:
+                outfile = arg
+        if not server or not outfile:
+            raise ValueError()
+
+    except Exception:
+        print 'USAGE: migrate-org-to-scheming.py ' \
+              '-s/--server <remote server> ' \
+              '-o/--output outfile.jsonl'
+        sys.exit(1)
+
+    output = codecs.open(outfile, 'w', encoding='utf-8')
+    ckan_instance = RemoteCKAN(server)
+    orgs = ckan_instance.call_action('organization_list',
+                                     {'all_fields': True})
+    for org in orgs:
+        if org.get('title'):
+            title_translated = org[u'title'].split(' | ')
```

Comment:
I wonder if we have some that weren't entered exactly like this. If so maybe splitting on `|` and stripping whitespace would be safer. Are there any orgs with no translations?

Reply:
https://github.com/open-data/ckanext-canada/blob/68e41fd6af164632b53f67cd4cf6109b879e0f39/transitional_orgs.jsonl#L110